### ISSUE TITLE
JS images

### ISSUE THREAD

**Issue by elvisef** [qsniyg/maxurl#884]
Can we make them larger here?
[https://p6podjarzebina.edupage.org/photos/?photo=album#](https://p6podjarzebina.edupage.org/photos/?photo=album#)

![2021-08-25_10-33](https://user-images.githubusercontent.com/25976749/130757230-086ffe7b-4577-465e-bf8a-1fa465525004.png)

If You click on each gallery there are few (or more) pictures.
But it is a kind of slider I think. Base images are thumnails.
Some FF img plugins detect only 640x480 but not original.
Only the right corner "cloud" download button can load the full image.

But.. if You slide each image, next next next... images links are being saved somewhere in the DOM I think, because if I slide some of them, or even all of them, the plugin like ImagePicka will get them all in appriopriate original versions.

But I can't make them larger with ImageMaxUrl. Maybe some options I need to set ?

**Comment by qsniyg** [comment]
I don't think it can be supported as a URL rule, as they appear to encrypt the directives.

It should be possible to support it as a host rule though. It dynamically inserts a piece of JS that contains the information stored as a JSON object, so all that's needed is to parse it. I'll work on it in a bit :)

**Comment by elvisef** [comment]
Would be nice : )
Because even if I try to use a plugin that gets images, it gets all images in the code, so galleries are mixed a bit, and generally there is a mess. I cannot recognize images needed for each gallery. Damn... horrible : ]
